packages/recommend/src/methods/getRecommendations.ts

```diff
@@ -0,0 +1,47 @@
+import { SearchOptions } from '@algolia/client-search';
+import { MethodEnum } from '@algolia/requester-common';
+
+import { RecommendClient, RecommendModel, WithRecommendMethods } from '../types';
+
+export type GetRecommendationsOptions = {
+  readonly indexName: string;
+  readonly model: RecommendModel;
+  readonly objectID: string;
+  readonly threshold?: number;
+  readonly maxRecommendations?: number;
+  readonly queryParameters?: SearchOptions;
+  readonly fallbackParameters?: Pick<SearchOptions, 'facetFilters'>;
+};
+
+type GetRecommendations = (
+  base: RecommendClient
+) => WithRecommendMethods<RecommendClient>['getRecommendations'];
+
+export const getRecommendations: GetRecommendations = base => {
+  return (options, requestOptions) => {
+    const requests: readonly GetRecommendationsOptions[] = [
+      {
+        // @TODO: we temporarly need to provide `queryParameters` because the
+        // endpoint doesn't default it yet.
```

Comment:
could we add a ticket number / GitHub issue here? or is there only private info available?

Reply:
It's more that the server crashes because the engine assume that `queryParameters` always exists. The engine team told me them they were fixing this asap. I'm in touch with them to remove this line of code as soon as it's not necessary anymore.

Reply:
It's fixed! See 8c8c3030eeab0691686f40387e0dbf3436874e35.